rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -0,0 +1,206 @@
+            amount = asset_normalized_value(
+                amount=hex_or_bytes_to_int(tx_log.data),
+                asset=token,
+            )
+            notes = f'Withdraw {amount} {token.identifier} from AAVE V2'
+            if to != user:
+                notes += f' to {to}'
+            for event in decoded_events:
+                if (
+                    event.balance.amount == amount and
+                    event.event_subtype == HistoryEventSubType.NONE
+                ):
+                    if (
+                        event.location_label == to and
+                        event.event_type == HistoryEventType.RECEIVE
+                    ):
+                        event.event_type = HistoryEventType.WITHDRAWAL
+                        event.notes = notes
+
+                    event.counterparty = CPT_AAVE_V2
+
+        elif tx_log.topics[0] == BORROW:
+            token = EvmToken(evm_address_to_identifier(
+                address=hex_or_bytes_to_address(tx_log.topics[1]),
+                token_type=EvmTokenKind.ERC20,
+                chain_id=self.ethereum.chain_id,
+            ))
+            on_behalf_of = hex_or_bytes_to_address(tx_log.topics[2])
+            user = hex_or_bytes_to_address(tx_log.data[:32])
+            amount = asset_normalized_value(
+                amount=hex_or_bytes_to_int(tx_log.data[32:64]),
+                asset=token,
+            )
+            rate_mode = hex_or_bytes_to_int(tx_log.data[64:96])
+            rate = token_normalized_value_decimals(
+                token_amount=hex_or_bytes_to_int(tx_log.data[96:128]),
+                token_decimals=25,
+            )
+            notes = f'Borrow {amount} {token.identifier} from AAVE V2 with {"stable" if rate_mode == 1 else "variable"} APY {rate}'  # noqa: E501
+            for event in decoded_events:
+                if (
+                    event.balance.amount == amount and
+                    event.event_subtype == HistoryEventSubType.NONE
+                ):
+                    if (
+                        event.location_label == user and
+                        event.event_type == HistoryEventType.RECEIVE and
+                        event.asset == token
+                    ):
+                        event.notes = notes
+
+                    event.counterparty = CPT_AAVE_V2
+
+        elif tx_log.topics[0] == REPAY:
+            token = EvmToken(evm_address_to_identifier(
+                address=hex_or_bytes_to_address(tx_log.topics[1]),
+                token_type=EvmTokenKind.ERC20,
+                chain_id=self.ethereum.chain_id,
+            ))
+            user = hex_or_bytes_to_address(tx_log.topics[2])
+            repayer = hex_or_bytes_to_address(tx_log.topics[3])
+            amount = asset_normalized_value(
+                amount=hex_or_bytes_to_int(tx_log.data),
+                asset=token,
+            )
+            notes = f'Repay {amount} {token.identifier} on AAVE V2'
```

Comment:
```suggestion
            notes = f'Repay {amount} {token.identifier} on AAVE v2'
```

and mention the user that is making the repayment